plutus-ledger-api/src/PlutusLedgerApi/V1/Value.hs

```diff
@@ -145,21 +170,28 @@ newtype AssetClass = AssetClass { unAssetClass :: (CurrencySymbol, TokenName) }
 assetClass :: CurrencySymbol -> TokenName -> AssetClass
 assetClass s t = AssetClass (s, t)
 
--- | A cryptocurrency value. This is a map from 'CurrencySymbol's to a
--- quantity of that currency.
---
--- Operations on currencies are usually implemented /pointwise/. That is,
--- we apply the operation to the quantities for each currency in turn. So
--- when we add two 'Value's the resulting 'Value' has, for each currency,
--- the sum of the quantities of /that particular/ currency in the argument
--- 'Value'. The effect of this is that the currencies in the 'Value' are "independent",
--- and are operated on separately.
---
--- Whenever we need to get the quantity of a currency in a 'Value' where there
--- is no explicit quantity of that currency in the 'Value', then the quantity is
--- taken to be zero.
---
--- See note [Currencies] for more details.
+{- | The 'Value' type represents a collection of amounts of different currencies.
+This is a map from 'CurrencySymbol's to a quantity of that currency.
```

Comment:
kind of, it's a map from currency symbols to token names to quantities. I wouldn't say it's directly a map to quantities. also I wouldn't describe it as a map ;) it's a finitely-supported function! or the vector space analogy is fine

Reply:
```suggestion
```